packages/databyss-documentation/website/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Hey, I had to move this file to the root of the package to facilitate the version fix script. Is there a way to keep it there?